src/atoms.ts

```diff
@@ -18,23 +20,25 @@ function formatOptionalDate(date: Int64 | undefined): string | undefined {
 function parseAtom(
     element: BlockElement,
     atoms: Atoms,
-    docParser: DocParser,
+    docParser: DocParser
```

Comment:
No trailing commas can result in a larger diff when changes are made - have we intentionally decided that we're fine with that? @sndrs 

This is interesting because a few lines below it's _added_ a trailing comma to an Object declaration 🤔

Reply:
hmmm htat doesn't seem right, agree trailing comma's should be there. wil lhave a look

Reply:
mentioned above, i think you need to use https://www.npmjs.com/package/@guardian/prettier too

Reply:
Thanks, I've added `@guardian/prettier` in this commit https://github.com/guardian/apps-rendering/pull/888/commits/a51475c64db5fa74e6e4930cf1faace99f3540f1